soccer/gameplay/plays/restarts/basic_direct.py

```diff
@@ -0,0 +1,57 @@
+import main
+import robocup
+import behavior
+import constants
+import enum
+
+import standard_play
+import evaluation
+import tactics.coordinated_pass
+import skills
+
+
+class BasicDirect(standard_play.StandardPlay):
+
+    #Minimum probability for which a shot should be taken
+    MIN_SHOT_PROB = 0.35
+    FIELD_LENGTH_MULTIPLIER = 0.75
+
+    class State(enum.Enum):
+        kicking = 1
+        passing = 2
+
+    def __init__ (self):
+        super().__init__(continuous = True)
+        for s in DirectFreeKick.State:
```

Comment:
Class is no longer called DirectFreeKick, so need to change names here and in behaviors